testy.js

```diff
@@ -65,11 +67,16 @@ class Testy {
   }
   
   _loadAllRequestedFiles() {
-    this._resolvedTestFilesPathsToRun().forEach(path =>
-      Utils.allFilesMatching(path, this._testFilesFilter()).forEach(file =>
-        require(file)
-      )
-    );
+    try{
+      this._resolvedTestFilesPathsToRun().forEach(path =>
+        Utils.allFilesMatching(path, this._testFilesFilter()).forEach(file =>
+          require(file)
+        )
+      );
+    }catch(err){
+      console.log(`${red}Error: the requested file/folder does not exist.`);
```

Comment:
I think it would be better to delegate this to the `ui` object (instance of `ConsoleUI`), I'd like to keep this object isolated from `console`